.vib/moodle/cypress/cypress/integration/moodle_spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+it('adds new course', () => {
+  cy.login();
+  cy.contains('a', 'Site administration').click();
+  cy.contains('a', 'Courses').click();
+  cy.contains('a', 'Add a new course').click();
+  cy.fixture('courses').then((courses) => {
+    cy.get('#id_fullname').type(courses.newCourse.fullName);
+    cy.get('#id_shortname').type(courses.newCourse.shortName);
+  });
+  cy.get('.fp-btn-add').click();
+  cy.contains('a', 'Upload a file').click();
```

Comment:
Could we omit the first parameter for this contains?

```suggestion
  cy.contains('Upload a file').click();
```

The same question for the others `cy.contains` used in this test.

Reply:
@mdhont I'm not sure if you reviewed the rest of `cy.contains` or that only this one could be changed, could you double-check it, please?

Reply:
Ok, I removed the rest as well